EnglishFilter/sections/foreign.txt

```diff
@@ -788,6 +788,14 @@ kliuki.bg###container > div[style="float:left;"] > div[style]
 ! https://github.com/AdguardTeam/AdguardFilters/issues/26330
 ||acs.youku.com/*.youku.play.$replace=/("ad":{).+"(\}\,"playlog")/\$1\$2/
 @@||cibntv.net/youku/*.mp4?$domain=v.youku.com
+! https://github.com/AdguardTeam/AdguardFilters/issues/32822
+zmz2019.com##.key-result > li > a[href*=".taobao.com"]
+||jstucdn.com/g3/
+.com/js/ad_$domain=zmz2019.com
```

Comment:
I suggest this rule as generic rule (EasyList China has it already as generic rule).
What do you think @Alex-302?